### PR TITLE
CY-2644 - Add documentation for Deployment Info widget

### PR DESCRIPTION
**Pull request by qooban**
Widget description available at:
https://docs.cloudify.co/staging/CY-2644/working_with/console/widgets/deploymentinfo/

### CODE REVIEW

File: content/working_with/console/widgets/deploymentInfo.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+![deployment-info]( /images/ui/widgets/deployment-info.png )
+
+#### Widget Settings
+* `Refresh time interval` - The time interval in which the widget’s data will be refreshed, in seconds. Default: 10 seconds
+* `Show blueprint name` - Defines whether deployment's blueprint name should be displayed. Default: On
+* `Show site name` - Defines whether deployment's site name should be displayed. Default: On
+* `Show created date` - Defines whether deployment's created date should be displayed
+* `Show updated date` - Defines whether deployment's last update date should be displayed. Default: On
+* `Show creator` - Defines whether deployment's creator username should be displayed. Default: On
+* `Show node instances status` - Defines whether deployment's node instances status should be displayed. Default: On
```
Comment (kubama) [inline content/working_with/console/widgets/deploymentInfo.md:28]: I don't know which style is correct (maybe both?) but there is inconsistency in apostrophes usage - see line 22 (widget`s) versus other lines (deployment's)

File: content/working_with/console/widgets/deploymentInfo.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+* **Site name**
+* **Creation time**
+* **Last update time**
+* **Creator username**
+
+
+![deployment-info]( /images/ui/widgets/deployment-info.png )
+
+#### Widget Settings
+* `Refresh time interval` - The time interval in which the widget’s data will be refreshed, in seconds. Default: 10 seconds
```
Comment (kubama) [inline content/working_with/console/widgets/deploymentInfo.md:22]:
```suggestion
* `Refresh time interval` - Time interval in which widget’s data will be refreshed, in seconds. Default: 10 seconds
```

File: content/working_with/console/widgets/deploymentInfo.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* `Show blueprint name` - Defines whether deployment's blueprint name should be displayed. Default: On
+* `Show site name` - Defines whether deployment's site name should be displayed. Default: On
+* `Show created date` - Defines whether deployment's created date should be displayed
```
Comment (kubama) [inline content/working_with/console/widgets/deploymentInfo.md:25]:
```suggestion
* `Show created date` - Defines whether deployment creation date should be displayed
```
or
```suggestion
* `Show created date` - Defines whether deployment's create date should be displayed
```

File: content/working_with/console/widgets/deploymentInfo.md (hunk 1 above)
Comment (qooban) [reply content/working_with/console/widgets/deploymentInfo.md]: Fixed.

**Pull request merged**